main.css

```diff
@@ -0,0 +1,34 @@
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;1,500&display=swap");
+@import url("css-files/alerts.css");
+@import url("css-files/avatars.css");
+@import url("css-files/badges.css");
+@import url("css-files/buttons.css");
+@import url("css-files/cards.css");
+@import url("css-files/inputs.css");
+@import url("css-files/modals.css");
+@import url("css-files/navigation-bars.css");
+@import url("css-files/ratings.css");
+@import url("css-files/toasts.css");
+@import url("css-files/typography.css");
+
+* {
```

Comment:
Instead of using the * selector here I think adding these styles to the body tag will be more synatctically correct.